lib/Backends/OpenCL/OpenCLDeviceManager.cpp

```diff
@@ -176,21 +171,38 @@ void OpenCLDeviceManager::addNetworkImpl(const Module *module,
     return;
   }
 
+  // Create a shared command queue using traceInfo.
```

Comment:
The comment is a bit misleading. The fact that is uses traceInfo is not very important, as it is optional.

It would be more interesting to describe who is sharing this command queue? Different runs of the same network? Different networks?